examples/bebop/bebop_test.cc

```diff
@@ -1,14 +1,14 @@
-// C++ includes
-#include <chrono>
-#include <iostream>
-#include <stdexcept>
-#include <thread>
-
 // BoB robotics includes
 #include "hid/joystick.h"
 #include "libbebop/bebop.h"
 #include "video/display.h"
 
+// Standard C++ includes
```

Comment:
How come you re-ordered this? If it's to overcome some inclusion-order horror maybe add a comment.

Reply:
Because I read somewhere that it's good to have your own includes at the top, so that if e.g. hid/joystick.h implicitly depends on one of the standard C++ includes you have underneath you'll get an error. Not sure if it's really a good idea or not